### PR TITLE
SC-5825 Removed Circular progress, added HTML5 progress and translated text

### PR DESCRIPTION
**Pull request by janrenz**
Removed Circular progress component that was not compatible with our security settings, added HTML5 progress, translated text
Bildschirmfoto 2020-10-14 um 10 57 17

### CODE REVIEW

File: locales/en.json [hunk 1]
```diff
@@ -1990,6 +1990,8 @@
 				"printSubmission": "Print submission",
 				"printTask": "Print task definition",
 				"submitted": "Submitted",
+				"statsSubmitted": "'{{submissionCount}}' out of {{totalCount}} submitted",
```
Comment (ABialoskorski) [inline locales/en.json:1993]: why in english version {{submissionCount}} and {{gradedCount}} are in quotes, but in german are not?

Comment (janrenz) [reply locales/en.json:1993]: fixed

**Comment by janrenz** [comment]
@ABialoskorski Can you review again please

File: views/homework/components/overview_card_template.hbs [hunk 2]
```diff
@@ -12,25 +12,39 @@
             {{$t "homework.text.noSubmissionDateSet"}}
             {{/if}}
         </div>
+        {{#if this.stats}}
+            <a class="content" href="{{this.url}}#activetabid=submissions"
+                aria-label="Abgaben der Aufgabe: '{{{escapeHtml this.name}}}'">
```
Comment (ABialoskorski) [inline views/homework/components/overview_card_template.hbs:17]: aria-label text should come from de/en.json

**Review by ABialoskorski — APPROVED** [review]

File: views/homework/components/overview_card_template.hbs (hunk 2 above)
Comment (janrenz) [reply views/homework/components/overview_card_template.hbs:17]: wasnot ther ebrefore the fix, should be a follow up.

**Pull request merged**